georgia/covid-viz.R

```diff
@@ -28,9 +28,10 @@ latest_ga$Residents.Initiated / latest_ga$Residents.Population
 
 ## read statewide prison data and create metrics for analysis 
 historical_statewide <- read_csv("https://raw.githubusercontent.com/uclalawcovid19behindbars/data/master/historical-data/historical_state_counts.csv")
-ga_prison_pop <- behindbarstools::read_mpap_pop_data() %>%
-    filter(State == "Georgia") %>%
-    pull(Population.Feb20)
+
+# http://104.131.72.50:3838/scraper_data/raw_files/2020-03-01_historical_ga_pop.pdf
+ga_prison_pop <- 55221
+
```

Comment:
Marshall Project data is from 4/1/2020 ([here](https://github.com/themarshallproject/COVID_prison_data/blob/master/data/prison_populations.csv)), not February 2020. Probably safer to use the number directly reported by the DOC, archived [here](http://104.131.72.50:3838/scraper_data/raw_files/2020-03-01_historical_ga_pop.pdf) for 3/1/2020 to have a safer source.

Reply:
yesss thank you for this!